server/routes/additionalConditions.js

```diff
@@ -15,7 +15,9 @@ module.exports = function({logger, conditionsService, licenceService}) {
         logger.debug('GET /additionalConditions');
 
         const nomisId = req.params.nomisId;
-        const conditions = await conditionsService.getAdditionalConditions();
+        const existingLicence = await licenceService.getLicence(req.params.nomisId);
```

Comment:
Only problem here is that your not handling errors that would be thrown from 
`await licenceService.getLicence(req.params.nomisId);`
and 
`await conditionsService.getAdditionalConditions(licence);`

It's ugly but you might want to wrap this code in a try catch :/